xhr.bs

```diff
@@ -1021,6 +1025,12 @@ method must run these steps:
   <p>Otherwise, if the <a>synchronous flag</a> is set, run these substeps:
 
   <ol>
+   <li>
+    <p>If <a>context object</a>'s <a>relevant settings object</a> has a
+    <a>responsible document</a> and it is <em>not</em> <a>allowed to use</a> the
```

Comment:
Should this be "and that"/"which is"? To me it seems that "it" refers to a settings object here.

Reply:
Fixed -- that was ambiguous, thanks.